test/integration/client/metrics/metrics_test.go

```diff
@@ -45,6 +46,7 @@ import (
 // regression test for https://issues.k8s.io/117258
 func TestAPIServerTransportMetrics(t *testing.T) {
 	featuregatetesting.SetFeatureGateDuringTest(t, feature.DefaultFeatureGate, "AllAlpha", true)
+	featuregatetesting.SetFeatureGateDuringTest(t, feature.DefaultFeatureGate, features.WatchList, false)
```

Comment:
For tests that enable all alpha and beta flags, we need to disable the WatchList FG until we can manually request progress notification from the server.

This is not an ideal solution because in the meantime someone might add a new test.

Reply:
We already can request progress notifications manually if you switch on ConsistentListFromCache feature
- this basically already works: https://github.com/kubernetes/kubernetes/pull/123513/files

So this shouldn't be needed - if it is it suggests something isn't working somewhere and we should debug it.

Reply:
ah, okay, I will have a look, thanks.